sources/models/fixtures/test-pem

```diff
@@ -0,0 +1 @@
MHlNVEEzTURjeU1UQXhNekJhRncweU5qQTNNRFl5TVRBeE16QmFNRGt4CkZUQVRCZ05WQkFNTURHSnZkSFJzWlhKdlkydGxkREVMTUFrR0ExVUVCaE1DVlZNeEV6QVJCZ05WQkFjTUNsZEIKVTBoSlRrZFVUMDR3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLQW9JQkFRQzErSk5MbllIeApyeGowOTJsQUpwOGRwRlRYbTVpR25jZFRubjBwem1OLzByMWhXUWNLSU9JY1UwcUdlN1h0UkVMcUFPQW5Gb3ZqCktaREV5SjB4eU9TZ1k4NFF6MVdXbUtqSHVsMGZXRHJ0RWlFNU03UklwVXBlVjZwejhOclRoOUFiSEZ1UWFMMkoKWk1COHVRWnFWRUhFZFl0THpFaHhnTU9NMUtjaXVqcjczWVE5TElLYVA3aW5sMU14YWd4c1Y4cjYyZG9WR3g2UwpsT2l3WXVQdmdkNnZWamQ5Wjl1UU5KNHFUdzBVVjBMbGJ1bXZwN2NITHg3U0EzWU55elZBWG0xb2FKYmhKeWpNClJmZEFwZDB3dkc0T1RXd2d5d240N2JPc0pBK3BJMDY0ZlNEeEpIaWxtdGVCSVRmV3hpQ1k2dkFvc01kVHJtRWwKUEFxYzVuem1WRkI3QWdNQkFBR2pVekJSTUIwR0ExVWREZ1FXQkJRZlFJYWh1cFRFMUdaeFBicnhYS3RoVitzTwovekFmQmdOVkhTTUVHREFXZ0JRZlFJYWh1cFRFMUdaeFBicnhYS3RoVitzTy96QVBCZ05WSFJNQkFmOEVCVEFECkFRSC9NQTBHQ1NxR1NJYjNEUUVCQ3dVQUE0SUJBUUFrc1BrNlhMTG5CdVRYK0xtYWJyK0VzcnNOQ2RBYitkOXkKc3Rpc2RDanBrbUtDc2VzSjk2UWdpMDYvMU04OHUvcCtHZHAreEhMUElKVUxyR2owMWoxdXJtRWJxd2lmeFVtawpmSkZ6bW00ZU56UVpqV1gyQm5HOVA4eHlHT1NWaWVHb1RFZW13MlBNT1hodUVFc0o5ckNQSXdzdlhMRzhOQW8yCnNjOGxaNXlOVnZkaHdwemdZL3lvM2pNKzJWeXZ4K2FnTXNhRGs3YmhjOTJGc3ZieTZ2a3ZweC82THZLSGFvSmgKbmxGcUwxcWZEZGZjOU81d1ZqeURyK3Q2NVBlVy9JN2ZCeXpZTEtyMDhIM3JCTzRQRXh0aVJrUHBJcGtEVWNnUAo3eTB6SGd6eVc5dHZNVlBXdXRDTDJ3UE1jb0hpdXJrRnNoOERtQzdzUytMMlNKTG1wbDJoCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0KYmFkCi0tLS0tQkVHSU4gQ0VSVElGSUNBVEUtLS0tLQpNSUlEVXpDQ0FqdWdBd0lCQWdJVVVDQW11NTFlNTQ2SGpKRlRHemdQaUtmcWV1VXdEUVlKS29aSWh2Y05BUUVMCkJRQXdPVEVWTUJNR0ExVUVBd3dNWW05MGRHeGxjbTlqYTJWME1Rc3dDUVlEVlFRR0V3SlZVekVUTUJFR0ExVUUKQnd3S1YwRlRTRWxPUjFSUFRqQWVGdzB5TVRBM01EY3lNVEF4TXpCYUZ3MHlOakEzTURZeU1UQXhNekJhTURreApGVEFUQmdOVkJBTU1ER0p2ZEhSc1pYSnZZMnRsZERFTE1Ba0dBMVVFQmhNQ1ZWTXhFekFSQmdOVkJBY01DbGRCClUwaEpUa2RVVDA0d2dnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUUMxK0pOTG5ZSHgKcnhqMDkybEFKcDhkcEZUWG01aUduY2RUbm4wcHptTi8wcjFoV1FjS0lPSWNVMHFHZTdYdFJFTHFBT0FuRm92agpLWkRFeUoweHlPU2dZODRRejFXV21Lakh1bDBmV0RydEVpRTVNN1JJcFVwZVY2cHo4TnJUaDlBYkhGdVFhTDJKClpNQjh1UVpxVkVIRWRZdEx6RWh4Z01PTTFLY2l1anI3M1lROUxJS2FQN2lubDFNeGFneHNWOHI2MmRvVkd4NlMKbE9pd1l1UHZnZDZ2VmpkOVo5dVFOSjRxVHcwVVYwTGxidW12cDdjSEx4N1NBM1lOeXpWQVhtMW9hSmJoSnlqTQpSZmRBcGQwd3ZHNE9UV3dneXduNDdiT3NKQStwSTA2NGZTRHhKSGlsbXRlQklUZld4aUNZNnZBb3NNZFRybUVsClBBcWM1bnptVkZCN0FnTUJBQUdqVXpCUk1CMEdBMVVkRGdRV0JCUWZRSWFodXBURTFHWnhQYnJ4WEt0aFYrc08KL3pBZkJnTlZIU01FR0RBV2dCUWZRSWFodXBURTFHWnhQYnJ4WEt0aFYrc08vekFQQmdOVkhSTUJBZjhFQlRBRApBUUgvTUEwR0NTcUdTSWIzRFFFQkN3VUFBNElCQVFBa3NQazZYTExuQnVUWCtMbWFicitFc3JzTkNkQWIrZDl5CnN0aXNkQ2pwa21LQ3Nlc0o5NlFnaTA2LzFNODh1L3ArR2RwK3hITFBJSlVMckdqMDFqMXVybUVicXdpZnhVbWsKZkpGem1tNGVOelFaaldYMkJuRzlQOHh5R09TVmllR29URWVtdzJQTU9YaHVFRXNKOXJDUEl3c3ZYTEc4TkFvMgpzYzhsWjV5TlZ2ZGh3cHpnWS95bzNqTSsyVnl2eCthZ01zYURrN2JoYzkyRnN2Ynk2dmt2cHgvNkx2S0hhb0poCm5sRnFMMXFmRGRmYzlPNXdWanlEcit0NjVQZVcvSTdmQnl6WUxLcjA4SDNyQk80UEV4dGlSa1BwSXBrRFVjZ1AKN3kwekhnenlXOXR2TVZQV3V0Q0wyd1BNY29IaXVya0ZzaDhEbUM3c1MrTDJTSkxtcGwyaAotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
Is `fixtures` a canonical name for a test data directory? In migrator I used `tests/data` because `tests` is the Rust built-in directory for 'integration' tests. I wonder how many variations of directory names we have for test input data.